static/js/src/advantage/subscribe/react/PurchaseModal.jsx

```diff
@@ -285,7 +285,7 @@ const PurchaseModal = () => {
               <>
                 <Summary />
                 {error && (
-                  <Notification type="negative" status="Error:">
+                  <Notification severity="negative" title="Error:">
```

Comment:
```suggestion
                  <Notification 
                      severity="negative"
                      title="Error:" 
                      inline
                  >
```

The notification needs to be inline 

Expected : 
![image](https://user-images.githubusercontent.com/11927929/128045818-c15c646f-8a9a-476d-bfa6-1fcb3fbf9b5e.png)

current : 
![image](https://user-images.githubusercontent.com/11927929/128045861-88dc13a3-8d34-4dcc-8f64-e52d2b1edc92.png)